src/components/ui/toolbox.ts

```diff
@@ -133,7 +135,7 @@ export default class Toolbox extends EventsDispatcher<ToolboxEvent> {
       items: this.toolsToBeDisplayed.map(tool => {
         return {
           icon: tool.toolbox.icon,
-          label: tool.toolbox.title,
+          label: I18n.t(I18nInternalNS.toolNames, tool.toolbox.title),
```

Comment:
what if user (or tool itself) does not specified the `toolbox.title`? The title is the optional field.

Please, test this case. Maybe there should be something like `tool.title || _.capitalize(tool.name)`